OpenOversight/tests/conftest.py

```diff
@@ -251,10 +251,15 @@ def add_mockdata(session):
 
     unit1 = models.Unit(descrip="test")
 
-    test_images = [models.Image(filepath='/static/images/test_cop{}.png'.format(x + 1), department_id=1) for x in range(5)] + \
-        [models.Image(filepath='/static/images/test_cop{}.png'.format(x + 1), department_id=2) for x in range(5)]
+    test_images = [
+        models.Image(
+            filepath=utils.get_presigned_image_url('/images/test_cop{}.png'.format(x + 1)),
+            department_id=1) for x in range(5)] + \
+        [models.Image(
+            filepath=utils.get_presigned_image_url('/images/test_cop{}.png'.format(x + 1)),
+            department_id=2) for x in range(5)]
```

Comment:
updated the `/static/...` image path to the normal path

Reply:
sweet, we're no longer hackily relying on two different routes between testing and production!